### PR TITLE
Add eslint

### PR DESCRIPTION
**Pull request by jentoobento**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,6 +14,8 @@
     "@react-navigation/drawer": "^5.8.5",
     "@react-navigation/native": "^5.7.0",
     "@react-navigation/stack": "^5.7.0",
+    "bufferutil": "^4.0.1",
+    "canvas": "^2.6.1",
```
Comment (enleiwu) [inline package.json:18]:
```suggestion
    "bufferutil": "^4.0.1",
    "canvas": "^2.6.1",
```

Are these packages being used?

**Comment by enleiwu** [comment]
Were you able to customize the linting rules? I'm getting the linting correction, but setting custom rules in the eslintric file didn't apply. (This isn't a big deal, but we might want to figure out why just in case if we need customize the rules in the future)

File: package.json (hunk 1 above)
Comment (jentoobento) [reply package.json:18]: I think it was a peer dependency

**Comment by jentoobento** [comment]
> Were you able to customize the linting rules? I'm getting the linting correction, but setting custom rules in the eslintric file didn't apply. (This isn't a big deal, but we might want to figure out why just in case if we need customize the rules in the future)

I haven't tried. Feel free to do whatever on my branch and push it up. Or approve this one, merge it, and make a new branch and test it there.

**Pull request merged**